glide.yaml

```diff
@@ -1,20 +1,20 @@
 package: go.uber.org/config
 import:
-- package: gopkg.in/validator.v2
-- package: gopkg.in/yaml.v2
 - package: github.com/pkg/errors
   version: ~0.8.0
 - package: golang.org/x/text
+  version: ~0.3.0
```

Comment:
Seeing as this is pre-1.0, this could potentially cause version mismatch errors from other libraries. We should take note of this in case it becomes an issue later.